phpstan.neon

```diff
@@ -0,0 +1,8 @@
+includes:
+    - vendor/landingi/php-coding-standards/phpstan.neon
+
+parameters:
+    reportUnmatchedIgnoredErrors: false
+    ignoreErrors:
+        - '#Property [a-zA-Z0-9\\]+::[a-zA-Z0-9\\\$]+ is never read, only written.#'
+        - '#Property [a-zA-Z0-9\\]+::[a-zA-Z0-9\\\$]+ is never written, only read.#'
```

Comment:
Możesz juz je usunąć, dodałem to ostatnio: https://github.com/landingi/php-coding-standards/pull/5/files